azure-devops/golang/stages/40-delivery/arm.yaml

```diff
@@ -78,6 +78,25 @@ stages:
               replaceExistingArchive: true
             displayName: 'Archive Build Output'
 
+          - task: AzureCLI@2
+            displayName: 'Getting Function Outbound IP Addresses'
+            inputs:
+              azureSubscription: '$(AZM_SERVICE_CONNECTION)'
+              scriptType: 'bash'
+              scriptLocation: 'inlineScript'
+              inlineScript: |
+                resourceGroupName=$(ENVIRONMENT)-rg-$(AZM_FUNCTION_NAME)-$(AZM_LOCATION)
```

Comment:
Are you able to re-use it as a variable? Because it's mentioned more than 1 time.

Reply:
Since this task is the first place where the 'resourceGroupName' is being used, I added the creation of the job variable in this task.